Anlab.Core/Models/TestItemPrices.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+using Anlab.Core.Domain;
+
+namespace Anlab.Core.Models
+{
+
+    public class TestItemPrices
+    {
+        public string Code { get; set; }
+        public decimal Cost { get; set; }
+        public decimal SetupCost { get; set; }
+        public string Name { get; set; }
+        public int Multiplier { get; set; }

```

Comment:
@jSylvestre As notes -- Setup Price is "WorkUnit" (aka multiplier) times SP-SetupPrice (aka SetupCost).  Probably worth adding some comments, and since we won't deal with either of those fields alone it could be worth adding a getter to determine setup price.